landing-page/scripts/tests/test_yolo_inference.py

```
@@ -0,0 +1,155 @@
+regardless of collection order; this doesn't disturb tasks_text_batch's
+already-imported reference to the stub's functions.
+"""
+import sys
+import types
+from pathlib import Path
+
+sys.path.insert(0, str(Path(__file__).resolve().parent.parent))
+
+if "models_api" not in sys.modules:
+    models_api_stub = types.ModuleType("models_api")
+    models_api_stub.get_model_file_path = lambda *a, **k: None
```

Ruff (0.16.1)

[warning] Unused lambda argument: `a`

(ARG005)